### PR TITLE
Add historical graphs card

### PR DESCRIPTION
**Pull request by nowNick**
Hi!

I've added a new card. However I think I'm gonna need a lot of feedback as I had to make some design decisions and I want to make sure they are in line with your vision for the app 😰 

Do you think the placement is correct or should I move it? What about the style of the graph? 
Is it ok that on mobile view you would see graphs first, and then "cities split" and "recent log"?

And... actually - well, do you think this card is useful and it's worth including?

Please let me know what you think 😉

### CODE REVIEW

File: src/components/Graphs/Graphs.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      const casesDateCount = sum(dataPerDate.filter(el => el.key === CASES_KEY))
+      const deathsDateCount = sum(dataPerDate.filter(el => el.key === DEATHS_KEY))
+      const curesDateCount = sum(dataPerDate.filter(el => el.key === CURES_KEY))
+
+      cumulativeCases += casesDateCount;
+      cumulativeDeaths += deathsDateCount;
+      cumulativeCures += curesDateCount;
+
+      return {
+        date: date.substring(5), // strip year
```
Comment (nowNick) [inline src/components/Graphs/Graphs.jsx]: I know it's hacky 🙈  ... but daylight saving time is coming and I didn't want to run into issues with parsing date and extracting month properly. Do you think it's ok?

**Comment by mhajder** [comment]
@nowNick In the dark mode I don't see any deaths in the tooltip.
image

**Comment by konradkalemba** [comment]
@mhajder It's also the case in other charts. We need to update tooltip formatting to be dependant on theme variables.

File: src/components/Graphs/Graphs.jsx (hunk 1 above)
Comment (konradkalemba) [reply src/components/Graphs/Graphs.jsx]: There is `moment.js` library installed you can use to parse date.

Comment (nowNick) [reply src/components/Graphs/Graphs.jsx]: Oh! That's great. I didn't want to add to the bundle size for such a simple use case but if it's already there then I'm gonna use it :)

**Comment by konradkalemba** [comment]
@nowNick Looks good! Currently I'm thinking about improving desktop layout, so we can show more statistics. Adding your graph in that spot in already "packed" layout isn't ideal. I'll find new place for your graph though. :)

**Comment by krzys1u** [comment]
Hello, when this Pull Request could be merged? i thing a lot of people is waiting for graph like this

**Comment by konradkalemba** [comment]
@krzys1u Currently, as I said, there is no enough room for this graph there. But today I will try to fit it into desktop layout I've been working on recently.

**Pull request merged**